x/evidence/types/evidence.go

```diff
@@ -76,7 +77,7 @@ func (e Equivocation) GetTotalPower() int64 { return 0 }
 // FromABCIEvidence converts a CometBFT concrete Evidence type to
 // SDK Evidence using Equivocation as the concrete type.
 func FromABCIEvidence(e comet.Evidence) *Equivocation {
-	bech32PrefixConsAddr := sdk.GetConfig().GetBech32ConsensusAddrPrefix()
+	bech32PrefixConsAddr := sdk.GetConfig().GetBech32ConsensusAddrPrefix() // TODO: remove this
```

Comment:
This is called from evidence beginBlocker which has access to the staking keeper, so why not do it now?

Reply:
modified thanks